Integrations/integration-FireEye_ETP.yml

```diff
@@ -0,0 +1,885 @@
+    as incidents.
+  name: message_status
+  defaultvalue: delivered (retroactive)
+  type: 0
+  required: false
+script:
+  script: |
+    from datetime import timedelta, datetime
+    import requests
+    import os
+    import re
+    import copy
+    import json
+    # disable insecure warnings
+    from requests.packages.urllib3.exceptions import InsecureRequestWarning
+    requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+    def set_proxies():
+        if demisto.params()['proxy']:
+            http = os.environ['http_proxy'] or os.environ['HTTP_PROXY']
+            https = os.environ['https_proxy'] or os.environ['HTTPS_PROXY']
+            proxies = {
+                'http': http,
+                'https': https
+            }
+            return proxies
+        return None
+
+
+    '''
+    GLOBAL VARS
+    '''
+
+    API_KEY = demisto.params().get('api_key')
+    PROXIES = set_proxies()
+    BASE_PATH = '{}/api/v1'.format(demisto.params().get('server'))
+    HTTP_HEADERS = {
+        'Content-Type': 'application/json'
+    }
+    USE_SSL = not demisto.params().get('unsecure')
+    MESSAGE_STATUS = demisto.params().get('message_status')
+
+    '''
+    SAERCH ATTRIBUTES VALID VALUES
+    '''
+
+    REJECTION_REASONS = [
+    'ETP102', 'ETP103', 'ETP104', 'ETP200', 'ETP201', 'ETP203','ETP204', 'ETP205',
+    'ETP300', 'ETP301', 'ETP302', 'ETP401','ETP402', 'ETP403', 'ETP404', 'ETP405']
+
+    STATUS_VALUES = ["accepted", "deleted", "delivered", "delivered (retroactive)", "dropped",
+    "dropped oob", "dropped (oob retroactive)", "permanent failure", "processing", "quarantined", "rejected", "temporary failure"]
+
+    '''
+    BASIC FUNCTIONS
+    '''
+    def listify(comma_separated_list):
+
+        if isinstance(comma_separated_list, list):
+            return comma_separated_list
+        return comma_separated_list.split(',')
+
+    def http_request(method, url, body=None, headers={}, url_params=None):
+        '''
+        returns the http response
+        '''
+
+        #add API key to headers
+        headers['x-fireeye-api-key'] = API_KEY
+
+        request_kwargs = {
+            'headers': headers,
+            'verify': USE_SSL,
+            'proxies': PROXIES
+        }
+
+        #add optional arguments if specified
+        if body is not None:
+            request_kwargs['data'] = json.dumps(body)
+        if url_params is not None:
+            request_kwargs['params'] = json.dumps(url_params)
+
+        LOG('attempting {} request sent to {} with body:\n{}'.format(method, url, json.dumps(body, indent=4)))
+        response = requests.request(
+            method,
+            url,
+            **request_kwargs
+        )
+
+        #handle request failure
+        if response.status_code not in range(200,205):
+            raise ValueError('Request failed with status code {}\n{}'.format(response.status_code, response))
```

Comment:
instead raising error lets use
```python
def return_error_and_exit(message):
    demisto.results({
        'Type': entryTypes['error'], # note, warning, error
        'ContentsFormat': formats['text'], # json, markdown, file, text
        'Contents': message
    })
    sys.exit(0)

return_error_and_exit('Request failed with status code {}\n{}'.format(response.status_code, response.text))

```

change `return_error_entry` to be `return_error_and_exit` so in the end it will exit